### PR TITLE
Fixed a bug that allowed to 2 times close/open report.

### PR DESCRIPTION
**Pull request by Vladimir-Urik**
Fixed a bug that allowed to 2 times close/open report.

### CODE REVIEW

File: modules/Core/pages/panel/users_reports.php [hunk 1]
```diff
@@ -285,24 +286,29 @@
...
+                            'commenter_id' => $user->data()->id,
+                            'comment_date' => date('Y-m-d H:i:s'),
+                            'date' => date('U'),
+                            'comment_content' => str_replace('{x}', Output::getClean($user->data()->username), $language->get('moderator', 'x_closed_report'))
+                        ));
+                        Session::flash('report_success', $language->get('moderator', 'report_closed'));
+                        Redirect::to(URL::build('/panel/users/reports/', 'id=' . Output::getClean($report[0]->id)));
+                    } else {
+
+                        Session::flash('report_failure', "You cannot close this report 2x");
```
Comment (Derkades) [inline modules/Core/pages/panel/users_reports.php:308]: Could you add a translation for this, just like the other Session::flash lines?

Comment (Vladimir-Urik) [reply modules/Core/pages/panel/users_reports.php]: > Could you add a translation for this, just like the other Session::flash lines?

Done.

PS: Sorry for a large number of commits 😅

Comment (Derkades) [reply modules/Core/pages/panel/users_reports.php]: don't worry, fortunately github has a squash option

**Comment by Aberdeener** [comment]
Thanks for the PR - How was this bug created or replicated in the first place?

**Comment by Derkades** [comment]
Regardless of whether this is a bug that can be reproduced using the UI, it's probably a good idea to prevent creating a new report when there already is one, for example if a malicious user decides to spam POST requests.

**Comment by Aberdeener** [comment]
Closing for inactivity

**Pull request closed without merging**